peribolos/knative-sandbox-OWNERS_ALIASES

```diff
@@ -114,16 +114,13 @@ aliases:
   - nainaz
   - salaboy
   - zroubalik
-  func-writers:
+  functastic-writers:
```

Comment:
For a lot of the other groups, we use the same reader/write/leads groups across both the knative/sandbox lists (so for example, we have serving-writers in both peribolos files, with the same names in each). Then you can also set up separate groups for sandbox repos if you want to allow someone to approve the sandbox but not the core repo ([example](https://github.com/knative-sandbox/container-freezer/blob/main/OWNERS#L4-L5)). Wonder if that might work here too (just thinking out loud, so feel free to disregard)?